src/positron-dts/positron.d.ts

```diff
@@ -1218,15 +1226,17 @@ declare module 'positron' {
 		 * @param focus Whether to focus the runtime's console
 		 * @param allowIncomplete Whether to bypass runtime code completeness checks. If true, the `code`
 		 *   will be executed by the runtime even if it is incomplete or invalid. Defaults to false
-		* @param mode Possible code execution modes for a language runtime
+		 * @param mode Possible code execution mode for a language runtime
+		 * @param errorBehavior Possible error behavior for a language runtime
```

Comment:
```suggestion
		 * @param errorBehavior Possible error behavior for a language runtime, currently ignored by kernels
```

Let's document this here too, in the Positron API for extensions.